### PR TITLE
Quickstart flutter mobile

### PR DESCRIPTION
**Pull request by sixhobbits**
Depends on https://github.com/ritza-co/fusionauth-quickstart-flutter-native

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205273462533583

### CODE REVIEW

**Comment by alex-fusionauth** [comment]
Moved to new Repo
https://github.com/FusionAuth/fusionauth-quickstart-flutter-native

File: astro/src/content/quickstarts/quickstart-flutter-native.mdx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+* The base URL of FusionAuth is `http://localhost:9011/`.
+
+You can log in to the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want to, but with Docker and Kickstart, everything will already be configured correctly.
+
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
+
+
+### Set Up A Public URL For FusionAuth
```
Comment (alex-fusionauth) [inline astro/src/content/quickstarts/quickstart-flutter-native.mdx:128]: Could we add some verbiage here to include this as an easy guide?
https://fusionauth.io/docs/v1/tech/developer-guide/exposing-instance

Then include this script
```shell
ngrok http --request-header-add 'X-Forwarded-Port:443' 9011
```

File: astro/src/content/quickstarts/quickstart-flutter-native.mdx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+Open the `main.dart` file in the `lib` directory of your project and paste the contents below into it.
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-flutter-native/main/complete-application/lib/main.dart"
+ lang="dart" />
+
+<Aside type="tip">
+Putting all your logic in one file makes sense for a tutorial but for a larger application, you'll probably want to split it up.
+</Aside>
+
+At the top of the file, change the `FUSIONAUTH_DOMAIN` constant to the public URL for your FusionAuth instance (the URL you used when configuring it).
```
Comment (alex-fusionauth) [inline astro/src/content/quickstarts/quickstart-flutter-native.mdx:222]: Include link back to `/#set-up-a-public-url-for-fusionauth` for clarity.

**Comment by rideam** [comment]
@alex-fusionauth I opened a PR here https://github.com/FusionAuth/fusionauth-quickstart-flutter-native/pull/1 to fix the rounding issue

**Comment by alex-fusionauth** [comment]
> @alex-fusionauth I opened a PR here [FusionAuth/fusionauth-quickstart-flutter-native#1](https://github.com/FusionAuth/fusionauth-quickstart-flutter-native/pull/1) to fix the rounding issue

awesome, merged :D

**Review by alex-fusionauth — APPROVED** [review]

**Review by alex-fusionauth — CHANGES_REQUESTED** [review]
@sixhobbits can we get an additional page added to the app?

We typically have an account page and make-change page when a user is logged in to show that you can secure both.

**Comment by rideam** [comment]
@alex-fusionauth I opened a PR https://github.com/FusionAuth/fusionauth-quickstart-flutter-native/pull/2 to add separate account and make-change pages

**Comment by alex-fusionauth** [comment]
> @alex-fusionauth I opened a PR [FusionAuth/fusionauth-quickstart-flutter-native#2](https://github.com/FusionAuth/fusionauth-quickstart-flutter-native/pull/2) to add separate account and make-change pages

Sorry missed this @rideam. Looks good and merged.

**Review by alex-fusionauth — APPROVED** [review]

File: astro/src/content/quickstarts/quickstart-flutter-native.mdx [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+After the installation process completes, you will see that the `flutterdemo` directory contains all the Flutter starter app configuration. Open the project directory with the text editor of your choice.
+
+You can run your new project on the actual device or an emulator to confirm everything is working before you customize any code. Do so by running the following command in the project directory.
+
+```shell
+cd flutterdemo
+flutter run
+```
+
+From the list of emulators or devices that is returned, choose the one you want the code to run in. Stay in this directory for the rest of this tutorial.
```
Comment (escii) [inline astro/src/content/quickstarts/quickstart-flutter-native.mdx:165]:
```suggestion
From the list of emulators or devices that can be used, choose the one that you want to run the code on.
Stay in this directory for the rest of this tutorial.
```

File: astro/src/content/quickstarts/quickstart-flutter-native.mdx [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+The login flow will look like this after FusionAuth is introduced.
+
+<LoginAfter alt={"Request flow during login after FusionAuth"}/>
+
+In general, you would introduce FusionAuth to normalize and consolidate user data, making it consistent and up-to-date and offloading your login security and functionality to FusionAuth.
+
+
+## Getting Started
+
+Start with getting FusionAuth up and running and creating a Flutter application.
```
Comment (escii) [inline astro/src/content/quickstarts/quickstart-flutter-native.mdx:78]:
```suggestion
Start by getting FusionAuth up and running and creating a Flutter application.
```

File: astro/src/content/quickstarts/quickstart-flutter-native.mdx [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+
+In that case, the system might look like this before FusionAuth is introduced.
+
+<LoginBefore alt={"Request flow during login before FusionAuth"}/>
+
+The login flow will look like this after FusionAuth is introduced.
+
+<LoginAfter alt={"Request flow during login after FusionAuth"}/>
+
+In general, you would introduce FusionAuth to normalize and consolidate user data, making it consistent and up-to-date and offloading your login security and functionality to FusionAuth.
```
Comment (escii) [inline astro/src/content/quickstarts/quickstart-flutter-native.mdx:73]:
```suggestion
In general, you would introduce FusionAuth to normalize and consolidate user data, ensuring consistency and reliable, while offloading your login security and functionality to FusionAuth.
```

**Review by escii — COMMENTED** [review]
This guide is definitely not for beginners. Prior mobile development knowledge is essential, and running it in emulators demands a substantial amount of RAM. Overall, it's a great guide. Just found some possible gramatical changes.

**Pull request merged**